JDBC/DatabasesPractice/src/main/java/com/laba/solvd/database/DAO/ICreatableMultiRelationship.java

```diff
@@ -0,0 +1,6 @@
+package com.laba.solvd.database.DAO;
+
+public interface ICreatableMultiRelationship<A, B, C> {
+
+	public void create(A a, B b, C c);

```

Comment:
in an interface, all methods and fields are implicitly public and abstract. You don't have to explicitly include it.